api/Performance.json

```diff
@@ -1073,7 +1073,7 @@
               "version_added": "53"
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
`performance.timeOrigin` confirmed not support in IE11 and Edge 15 using https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8471